bug: ci/deploy job fails with error

After https://github.com/coder/coder/issues/7884 the ci/deploy job has been failing with the following error:

```
cli/server_slim.go:33:123: cliui.Styles.Code undefined (type cliui.Styles has no method Code)
```

Examples:
- https://github.com/coder/coder/actions/runs/5196307682/jobs/9369878842
- https://github.com/coder/coder/actions/runs/5197433494/jobs/9372275425

--- a/cli/root.go
+++ b/cli/root.go
@@ -687,7 +687,7 @@ func formatExamples(examples ...example) string {
 				"  - " + padStyle.Render(e.Description + ":")[4:] + "\n\n    ",
 			)
 		}
-		// We add 1 space here because `cliui.Styles.Code` adds an extra
+		// We add 1 space here because `cliui.DefaultStyles.Code` adds an extra
 		// space. This makes the code block align at an even 2 or 6
 		// spaces for symmetry.
 		_, _ = sb.WriteString(" " + cliui.DefaultStyles.Code.Render(fmt.Sprintf("$ %s", e.Command)))

--- a/cli/server_slim.go
+++ b/cli/server_slim.go
@@ -30,7 +30,7 @@ func (r *RootCmd) Server(_ func(context.Context, *coderd.Options) (*coderd.API, 
 }
 
 func serverUnsupported(w io.Writer) {
-	_, _ = fmt.Fprintf(w, "You are using a 'slim' build of Coder, which does not support the %s subcommand.\n", cliui.Styles.Code.Render("server"))
+	_, _ = fmt.Fprintf(w, "You are using a 'slim' build of Coder, which does not support the %s subcommand.\n", cliui.DefaultStyles.Code.Render("server"))
 	_, _ = fmt.Fprintln(w, "")
 	_, _ = fmt.Fprintln(w, "Please use a build of Coder from GitHub releases:")
 	_, _ = fmt.Fprintln(w, "  https://github.com/coder/coder/releases")

--- a/enterprise/cli/proxyserver_slim.go
+++ b/enterprise/cli/proxyserver_slim.go
@@ -29,7 +29,7 @@ func (r *RootCmd) proxyServer() *clibase.Cmd {
 }
 
 func serverUnsupported(w io.Writer) {
-	_, _ = fmt.Fprintf(w, "You are using a 'slim' build of Coder, which does not support the %s subcommand.\n", cliui.Styles.Code.Render("server"))
+	_, _ = fmt.Fprintf(w, "You are using a 'slim' build of Coder, which does not support the %s subcommand.\n", cliui.DefaultStyles.Code.Render("server"))
 	_, _ = fmt.Fprintln(w, "")
 	_, _ = fmt.Fprintln(w, "Please use a build of Coder from GitHub releases:")
 	_, _ = fmt.Fprintln(w, "  https://github.com/coder/coder/releases")

